pages/properties/gatsby.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Gatsby",
+    "creator": "GitHub Actions",
```

Comment:
We don't really need the `creator` if GitHub is the creator. That is set by default.